Mathlib/LinearAlgebra/AffineSpace/FiniteDimensional.lean

```diff
@@ -217,6 +230,25 @@ lemma AffineIndependent.card_le_finrank_succ [Fintype ι] {p : ι → P} (hp : A
   exact (affineIndependent_iff_le_finrank_vectorSpan _ _
     (tsub_add_cancel_of_le <| Nat.one_le_iff_ne_zero.2 Fintype.card_ne_zero).symm).1 hp
 
+open Finset in
+/-- If an affine independent finset is contained in the affine span of another finset, then it is
+smaller than that finset. -/
```

Comment:
"smaller" to me implies `<`; can you clarify here?

Is the strict inequality true as well, with a strong `hst`?

Reply:
I've added the strict inequality